administration/src/bp-modules/cards/CreateCardsForm.tsx

```diff
@@ -52,10 +62,15 @@ interface Props {
 
 const CreateCardsForm = (props: Props) => {
   const { cardBlueprints, setCardBlueprints, region } = props
+  const bottomRef = useRef<HTMLDivElement>(null)
   const projectConfig = useContext(ProjectConfigContext)
   const [isModified, setModified] = useState(false)
 
   const addForm = () => {
+    if (bottomRef.current) {
+      console.log('scroll')
+      bottomRef.current!.scrollIntoView({ behavior: 'smooth' })
```

Comment:
you could just use the `scrollToBottom` fkt from below instead of duplicating the code